Modules/UDP/.travis.yml

```diff
@@ -0,0 +1,16 @@
+notifications:
```

Comment:
Could you replace this with the updated version such as [here](https://github.com/Zewo/Zewo/blob/master/Modules/Core/.travis.yml)? A few minor differences